src/main/resources/runner/environments/reg.env

```
@@ -1,6 +1,6 @@
-PF_VIR_MEM=50000
+PF_VIR_MEM=377850
```

I like round numbers 😄, unless there's a reason for this specific number.
```suggestion
PF_VIR_MEM=400000
```